CONTRIBUTING.md

```diff
@@ -158,21 +185,23 @@ with your recently pushed changes. You can also open a pull request from the
 Select your fork and your 'my_feature' branch, and open a pull request against
 the 'main' branch.
 
-2. There is now a checklist of items that need to be done before your PR can be merged.
-	* The continuous integration (CI) tests must complete and pass. The CI
-	runs several different checks including running the unit tests, ensuring
-	the documentation builds, checking for code style issues (see the [PEP8](https://peps.python.org/pep-0008/) style guide),
-	and ensuring any changes are covered by unit tests. The CI runs upon opening
-	a PR, and will re-run any time you push commits to that branch.
-	* Our code style checker uses [ruff](https://docs.astral.sh/ruff/), and using
-	`ruff check` locally can be helpful to identify style issues before opening a PR.
-	* You will need to add a change log entry in changes/PRID.fragmenttype.rst
+2. Ensure the GitHub CI checks are all passing.
+    * Unit tests will be run, and unit test coverage will be checked for
+	any new code. See [Writing and Running Unit Tests](#writing-and-running-unit-tests)
+    to learn how to run and debug the unit tests if you're seeing failures.
+	* Various code style checks will be run (see [Code Style](#code-style)). The
+	pre-commit hook described above is helpful for catching and debugging these locally.
+	* You will need to add a change log entry in `changes/PRID.fragmenttype.rst`
 	if your contribution is a new feature or bug fix.
 	An entry is not required for small fixes like typos.
-	* Your PR will need to be reviewed and approved by at least two maintainers.
-	They may require changes from you before your code can be merged, in which
-	case you will need to go back and make these changes and push them (they will
-		automatically appear in the PR when they're pushed to origin/my_feature).
+
+3. Ensure all the items in the **Tasks** checklist on the PR are completed.
+Instructions for how to do this are included in the checklist itself.
+
+4. Your PR will need to be reviewed and approved by at least two maintainers.
```

Comment:
This is news to me. When did PRs start requiring 2 approvals?

Reply:
I don't know and I didn't write it but I can change it now.

Reply:
Thanks. I didn't see that it existed previously. I'd say leave it and it can be reworded in a follow-up PR (to tease apart the maintainer vs developer differences).